src/main/java/com/minecolonies/coremod/colony/jobs/JobDeliveryman.java

```diff
@@ -179,6 +171,32 @@ public void serializeToView(final PacketBuffer buffer)
         return getDataStore().getQueue();
     }
 
+    @Override
+    public int getInactivityLimit()
+    {
+        return 60 * 10;
```

Comment:
magic numbers :)

Reply:
Well, those are explained by the javadoc, that is time in seconds, so that's 10 minutes. The -1 makes sense to be a constant though

Reply:
we already do have time constants for the 60 purpose though